helm/botkube/templates/deployment.yaml

```diff
@@ -113,6 +113,14 @@ spec:
               value: "{{.Release.Namespace}}"
             - name: BOTKUBE_SETTINGS_LIFECYCLE__SERVER_DEPLOYMENT_NAME
               value: "{{ include "botkube.fullname" . }}"
+            {{- if .Values.config.source.endpoint }}
+            - name: CONFIG_SOURCE_ENDPOINT
+              value: {{ .Values.config.source.endpoint }}
+            {{- end }}
+            {{- if .Values.config.source.identifier }}
+            - name: CONFIG_SOURCE_IDENTIFIER
```

Comment:
Once we change values, can we also rename these envs to `CONFIG_PROVIDER_ENDPOINT` and `CONFIG_PROVIDER_IDENTIFIER`?